File: app/src/main/java/com/ishow/noah/modules/main/tab2/Tab2Fragment.java
package com.ishow.noah.modules.main.tab2;

import android.os.Bundle;
import androidx.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.ishow.noah.R;
import com.ishow.noah.modules.base.AppBaseFragment;
import com.ishow.common.widget.StatusView;

/**
 * Created by yuhaiyang on 2017/4/21.
 * Home Fragment
 */

public class Tab2Fragment extends AppBaseFragment {

    private View mRootView;

    public static Tab2Fragment newInstance() {

        Bundle args = new Bundle();

        Tab2Fragment fragment = new Tab2Fragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        if (mRootView != null) {
            return mRootView;
        }

        mRootView = inflater.inflate(R.layout.fragement_tab_2, container, false);

        StatusView statusView = (StatusView) mRootView.findViewById(R.id.status_view);
        statusView.showLoading();
        return mRootView;
    }
}

File: app/src/main/java/com/ishow/noah/modules/main/tab2/Tab2Fragment.kt
package com.ishow.noah.modules.main.tab2

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.ishow.noah.R
import com.ishow.noah.databinding.FTab2Binding
import com.ishow.noah.modules.base.mvvm.view.AppBindFragment

/**
 * Created by yuhaiyang on 2020-05-11.
 */
class Tab2Fragment : AppBindFragment<FTab2Binding, Tab2ViewModel>() {

    override fun getLayout(): Int = R.layout.f_tab2

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return super.onCreateView(inflater, container, savedInstanceState)
    }

    companion object {

        fun newInstance(): Tab2Fragment {

            val args = Bundle()

            val fragment = Tab2Fragment()
            fragment.arguments = args
            return fragment
        }
    }
}